src/main/java/it/pagopa/ecommerce/commons/client/NpgClient.java

```diff
@@ -590,6 +675,12 @@ private CreateHostedOrderRequestDto buildOrderRequestDto(
                                 .resultUrl(resultUrl.toString())
                                 .cancelUrl(cancelUrl.toString())
                                 .notificationUrl(notificationUrl.toString())
+                                .recurrence(
+                                        contractId != null ? new RecurringSettingsDto()
+                                                .action(RecurringActionDto.SUBSEQUENT_PAYMENT)
```

Comment:
this has to be set for every payment with onboarded methods? also if it's the first payment?